java/src/jmri/jmrix/can/cbus/CbusPredefinedMeters.java

```diff
@@ -129,7 +147,8 @@ public void enable() {
             tc.addCanListener(CbusPredefinedMeters.this);
             log.info("Enabled meter Long Ex2Data {} {}", 
                 new CbusNameService(_memo).getEventNodeString(_nodeToListen,_eventToListenCurrent), 
-                new CbusNameService(_memo).getEventNodeString(_nodeToListen,_eventToListenVoltage));
+                new CbusNameService(_memo).getEventNodeString(_nodeToListen,_eventToListenVoltage),
+                new CbusNameService(_memo).getEventNodeString(_nodeToListen,_eventToListenCurrentExtra));
```

Comment:
You have only two `{}` in the log statement, but three parameters to the log.info call. Is this correct?

Reply:
Fixed